### PR TITLE
Added currently playing player queues to widget row

### PR DESCRIPTION
**Pull request by jozefKruszynski**

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -108,6 +159,13 @@ const widgetRows = ref<Record<string, WidgetRow>>({
 });
 
 onMounted(async () => {
+  const unsub = api.subscribe(EventType.PLAYER_UPDATED, (evt: EventMessage) => {
+    // signal user that there might be updated info available for this item
+    queueWdiget();
```
Comment (marcelveldt) [inline src/views/HomeView.vue]: typo ?

File: src/translations/en.json [hunk 2]
```diff
@@ -352,6 +352,7 @@
     "volume_normalization": "Volume normalization",
     "play_now_replace": "Play Now (clear queue)",
     "currently_playing": "Currently playing",
+    "currently_queued": "Currently queued",
```
Comment (marcelveldt) [inline src/translations/en.json]: I would really change this to currently playing

File: src/views/HomeView.vue (hunk 1 above)
Comment (jozefKruszynski) [reply src/views/HomeView.vue]: Ha. I guess so

File: src/translations/en.json (hunk 2 above)
Comment (jozefKruszynski) [reply src/translations/en.json]: Fair enough, will do

File: src/views/HomeView.vue [hunk 3]
```diff
@@ -196,6 +255,26 @@ onMounted(async () => {
   });
 });
 
+const queueWdiget = function () {
```
Comment (marcelveldt) [inline src/views/HomeView.vue]: net to the typo, I would rename this to make it more clear what it does.
Something like "updateNowPlayingWidget" or whatever

Comment (jozefKruszynski) [reply src/views/HomeView.vue]: Me and my "temporary" names for things 🤦‍♂️

**Review by marcelveldt — APPROVED** [review]
Nice job!

**Pull request merged**